docs/src/manual.md

```diff
@@ -161,6 +161,38 @@ You can solve a primal problem by using its dual formulation using the `DualOpti
 DualOptimizer
 ```
 
+Solving an optimization problem via its dual representation can be useful because some conic solvers assume the model is in the standard form and others use the geometric form.
+
+Standard form has affine expressions in cones
```

Comment:
This is the opposite. Below is the geometric form

Reply:
My bad hehehe